src/actions/commands/embedBuilder/embedEdit.ts

```
@@ -0,0 +1,842 @@
+  ) {
+    switch (selectedOption) {
+      case "content":
+        this.embedData.title =
+          modalSubmit.fields.getTextInputValue("title") || this.embedData.title;
+        this.embedData.description =
+          modalSubmit.fields.getTextInputValue("description") || undefined;
+        this.embedData.url =
+          modalSubmit.fields.getTextInputValue("url") || undefined;
+
+        const colorInput = modalSubmit.fields.getTextInputValue("color");
+        if (colorInput) {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)